### DISCUSSION TITLE
helm-charts

### DISCUSSION THREAD

**Discussion by agoldis** [sorry-cypress/sorry-cypress#256] [Ideas]
@tico24 I have merged your PR! 🎉

1. An undesired side effect is github release and tag creation (I guess by "release" action)

- https://github.com/sorry-cypress/sorry-cypress/releases/tag/sorry-cypress-0.1.1
- https://github.com/sorry-cypress/sorry-cypress/tags

May be using https://github.com/sorry-cypress/charts would eliminate this problem

2. Should https://github.com/sorry-cypress/sorry-cypress/blob/master/charts/sorry-cypress/Chart.yaml#L14 be updated with the most recent version on each new release of the source code?

3. What's the purpose of https://github.com/sorry-cypress/sorry-cypress/blob/master/charts/test.yaml ? Not sure it's being used

4. Now, with the changes merged, can we somehow make sure that the test is actually working?

And thanks again for the awesome work!

**Comment by tico24** [comment]
1: That's the helm release and is the desired outcome. I could probably improve the name for clarity.

2: Yes ideally. The chart version needs to increase too, you can't re-create chart versions like you can with docker tags.

3: I agree, it doesn't appear to be necessary any longer.

4: see PR #257 . I have checked... it doesn't. I'm going to cry in the corner for a bit now!

**Comment by agoldis** [comment]
haha 😂 I am off for today, will continue tomorrow.
I am ok to skip the test if it makes you sad, but I am truly intrigued by the mystery.

**Comment by tico24** [comment]
I forgot to say for 2, yes this would solve the problem and would be cleaner.
